react/features/poll/actions.js

```diff
@@ -0,0 +1,74 @@
+import { POLLS_PANE_CLOSE, POLLS_PANE_OPEN, NEW_POLL, NEW_POLL_RESPONSE,
```

Comment:
Can we alphabetize these imports? https://github.com/jitsi/jitsi-meet/blob/master/CONTRIBUTING.md#formatting